packages/web/src/common/store/account/sagas.js

```diff
@@ -130,44 +127,6 @@ function* onSignedIn({ payload: { account } }) {
       feePayerOverride
     })
   }
-
-  // Repair users from flare-101 that were impacted and lost connected wallets
```

Comment:
Is this removed because there's no longer necessary or something that needs to change because of this upgrade?

Reply:
No longer necessary and `disconnected_wallet_fix.json` was adding 50kb to the bundle!